src/CheckoutSdk/Payments/ThreeDSRequest.cs

```diff
@@ -18,5 +18,8 @@ public class ThreeDsRequest
         public string Version { get; set; }
 
         public Exemption? Exemption { get; set; }
+
+        [JsonProperty(PropertyName = "challenge_indicator")]
```

Comment:
This is not necessary. The current serialization make this change automatically.